### ISSUE TITLE
Always Says Only one URL is allowed

### ISSUE THREAD

**Issue by bendangelo** [ohler55/perfer#6]
This command: `./bin/perfer --threads 4 --connections 20 --path index.html http://locahost:3000`

I get this response:

```
*-*-* Only one URL is allowed.                                                         

usage: ./bin/perfer [options] <server-address>           
...
```

I'm following the docs. This should run the benchmark right?

**Comment by ohler55** [comment]
I suspect the reason is that since there is no `--path` option perfer treats that as a URL. Try it by moving the index.html to be included in the URL.

**Comment by bendangelo** [comment]
I tried that as well, I get the same message

**Comment by ohler55** [comment]
It looks like it does not handle `--connections` correctly. I'll fix that. In the mean time `-c` does work. Thanks for reporting this.

**Comment by ohler55** [comment]
The issue was a simple one. I left the 's' off of "connections" I'll get rid of some or all the warnings that have appeared over the years and make a release.

**Comment by ohler55** [comment]
Released v1.5.4.

**Comment by bendangelo** [comment]
thanks
